### DISCUSSION TITLE
Dom elements in vanX.reactive?

### DISCUSSION THREAD

**Discussion by HEAVYPOLY** [vanjs-org/van#331] [Q&A] [answered]
```js 
const inp = vanX.reactive({
  mode: 'idle',
  target: document.body,
})
  
  ```
  
  are dom elements allowed in vanX.reactive like this?

**Comment by sirenkovladd** [comment, marked as answer]
It doesn't seem to work
https://jsfiddle.net/kf52c1ox/16/

You can read `why?`
https://vanjs.org/advanced#why-not-dom-valued-states

**Comment by HEAVYPOLY** [comment]
Thank you!
